File: src/Samples/Client_Server/REST_WebClient/REST_ToDoItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DotNetForHtml5.Showcase.SampleRestWebService.Models
{
    public class ToDoItem
    {
        public ToDoItem() { }

        public ToDoItem(string description, int ownerId)
        {
            Description = description;
        }

        public Guid Id { get; set; }
        public Guid OwnerId { get; set; }
        public string Description { get; set; }
    }
}

File: src/Samples/Client_Server/REST_WebClient/REST_ToDoItem.fs
namespace OpenSilver.Samples.Showcase.SampleRestWebService.Models

open System.Runtime.Serialization

[<DataContract(Name = "ToDoItem", Namespace = "http://schemas.datacontract.org/2004/07/DotNetForHtml5.Showcase.SampleRestWebService.Models")>]
type ToDoItem = {
    [<field: DataMember(Name = "Description")>]
    Description: string
    [<field: DataMember(Name = "Id")>]
    Id: string
    [<field: DataMember(Name = "OwnerId")>]
    OwnerId: string
}
